contents/docs/tutorials/revenue.md

```diff
@@ -0,0 +1,227 @@
+        event="purchase"
+        properties={
+            "plan": sale_data["plan_name"],
+            "price": sale_data["price"],
+            "currency": sale_data["currency"]
+        },
+    )
+```
+
+In the example above, `_sale_completed` would be called after your existing logic for purchases determines a sale has been successfully completed. An event will then be sent to PostHog with the information you want to include about the sale, such as the plan, the price, the currency, and anything else you may find relevant. 
+
+> **Note:** The most important thing here is to ensure you only send the event to PostHog when you are sure a sale completed. This ensures your tracking data matches your actual revenue, preventing an event from being captured for a sale that later gets cancelled. 
+
+### Visualizing your revenue over time
+
+Having started to send events that track sales to PostHog, the first thing to do is visualize your revenue over time. This can be done in 'Trends', and might look something like this:
+
+![Revenue Over Time Chart](../../images/tutorials/revenue/revenue-over-time.png)
+
+The chart above is targeting a `$purchase` event over the time period of "Last 90 days". However, instead of tracking the total volume of the event, it is tracking a sum of the value of one of its properties, called `purchase_value`.
+
+The way you can achieve this view is by clicking on the dropdown to the right of the property name, which is set to `Total volume` by default. Once selected, this will reveal the following options:
+
+- Total volume
+- Active users
+- Sum
+- Average
+- Maximum
+- Minimum
+
+The first two options target the event itself, showing either the volume of events or the number of active users who triggered the event. 
+
+However, the last four options target numerical properties on events. When tracking revenue, this allows you to visualize:
+
+- The total revenue per time period (e.g. by hour, day, or month)
+- The largest (and smallest) purchase per time period
+- The average purchase value per time period
+
+Here are is an example dashboard combining charts that use all four of the numerical operations:
+
+![Operations Dashboard Image](../../images/tutorials/revenue/operations-dashboard.png)
+
+
+### Total Revenue In The Last Month
+
+If you want to see agreggate values for periods, you can change the time frame on line charts, or you can also display data as a table or pie chart, which combines the aggregate values of the periods and displays it as one value. 
+
+To see the total revenue of the previous month, for instance, you can set your time frame to 'Previous Month' and your chart type to 'Pie'. You can pair this with a view of this month's total month-to-date revenue, as well as compare performance on a week-by-week basis. 
+
+Here's what a dashboard with these views might look like:
+
+![TR Per Month Image](../../images/tutorials/revenue/tr-month.png)
+
+### Breaking Down Revenue By Source
```

Comment:
As a marketer, I'm immediately like "GIVE ME THE ROI!"

Could we suggest a way to do this (there probably isn't a good API-friendly way, it'd be a brilliant plugin though to track spend data by UTM tag or whatever) so I imagine our answer may just be "you can then fire up your ad platform of choice to compare spend to revenue to work out your ROIs, or better yet, build a plugin to automate this (insert link)"

Reply:
Right, will add a note. 

Yeah the difficulty regarding the ROI calculation is that we'd need your cost in PostHog - and that's not too straightforward. Definitely a lot plugins can do here though.